addons/l10n_hu/data/account_tax_report_data.xml

```diff
@@ -1,189 +1,246 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
```

Comment:
Please remove the whitespace diff in this file (it's fine for the taxes).
While it is interesting to have it, we should do it in 15.1 where it  is truly implemented. In here, it just makes the diff harder to read and fixes harder to rebase through.